doc/sphinx/proof-engine/tactics.rst

```diff
@@ -2634,28 +2625,25 @@ as regular setoids for :tacn:`rewrite` and :tacn:`setoid_replace` (see
    with `U` providing that `U` is well-formed and that `T` and `U` are
    convertible.
 
-.. exn:: Not convertible.
-
+   .. exn:: Not convertible.
 
-.. tacv:: change @term with @term
+   .. tacv:: change @term with @term’
 
-  This replaces the occurrences of :n:`@term` by :n:`@term` in the current goal.
-  The term :n:`@term` and :n:`@term` must be convertible.
+      This replaces the occurrences of :n:`@term` by :n:`@term’` in the current goal.
+      The term :n:`@term` and :n:`@term’` must be convertible.
 
-.. tacv:: change @term at {+ @num} with @term
+   .. tacv:: change @term at {+ @num} with @term’
 
-  This replaces the occurrences numbered :n:`{+ @num}` of :n:`@term by @term`
-  in the current goal. The terms :n:`@term` and :n:`@term` must be convertible.
+      This replaces the occurrences numbered :n:`{+ @num}` of :n:`@term` by :n:`@term’`
+      in the current goal. The terms :n:`@term` and :n:`@term’` must be convertible.
 
-.. exn:: Too few occurrences.
+      .. exn:: Too few occurrences.
 
-.. tacv:: change @term in @ident
-.. tacv:: change @term with @term in @ident
-.. tacv:: change @term at {+ @num} with @term in @ident
+   .. tacv:: change @term {? {? at {+ @num}} with @term} in @ident
 
-  This applies the change tactic not to the goal but to the hypothesis :n:`@ident`.
+      This applies the :tacn:`change` tactic not to the goal but to the hypothesis :n:`@ident`.
 
-See also: :ref:`Performing computations <performingcomputations>`
+   .. seealso :: :ref:`Performing computations <performingcomputations>`
```

Comment:
is the space intended?

Reply:
After `seealso`? No, oops!

Reply:
For some reason, it didn't prevent Sphinx from rendering it correctly: https://coq.gitlab.io/-/coq/-/jobs/68113531/artifacts/_install_ci/share/doc/coq/sphinx/html/proof-engine/tactics.html#coq:tacn.change